core/src/test/java/org/apache/iceberg/rest/TestRESTUtil.java

```diff
@@ -212,4 +213,12 @@ public void namespaceFromQueryParam() {
     assertThat(RESTUtil.namespaceFromQueryParam("one%1Ftwo\u001fns"))
         .isEqualTo(Namespace.of("one%1Ftwo", "ns"));
   }
+
+  @Test
+  public void generateUuidV7_hasVersionAndVariant() {
```

Comment:
```suggestion
  public void uuidV7HasVersionAndVariant() {
```

Reply:
@huaxingao It seems the rename was accidentally reverted?